packages/app/src/store/studio-store.ts

```diff
@@ -206,6 +212,8 @@ export const useStudioStore = defineStore('studioRecorder', {
         }
       }
 
+      this.setUrl(getAutUrl())
```

Comment:
No need to do `getAutUrl` like this, we should be able to do

```js
const autStore = useAutStore()
this.setUrl(autStore.url)
```

Although I think there's a bug around this anyway, as Mike pointed out in Slack, there's some weirdness around the URL logic.